backend/baton/src/main/java/touch/baton/domain/member/Member.java

```diff
@@ -84,28 +84,58 @@ private void validateNotNull(final MemberName memberName,
                                  final Company company,
                                  final ImageUrl imageUrl
     ) {
-        if (Objects.isNull(memberName)) {
-            throw new MemberDomainException("Member 의 name 은 null 일 수 없습니다.");
-        }
+        validateMemberNameNotNull(memberName);
+        validateSocialIdNotNull(socialId);
+        validateOauthIdNotNull(oauthId);
+        validateGithubUrlNotNull(githubUrl);
+        validateCompanyNotNull(company);
+        validateImageUrlNotNull(imageUrl);
+    }
 
-        if (Objects.isNull(socialId)) {
-            throw new MemberDomainException("Member 의 socialId 은 null 일 수 없습니다.");
+    private static void validateImageUrlNotNull(final ImageUrl imageUrl) {
+        if (Objects.isNull(imageUrl)) {
+            throw new MemberDomainException("Member 의 imageUrl 은 null 일 수 없습니다.");
         }
+    }
 
-        if (Objects.isNull(oauthId)) {
-            throw new MemberDomainException("Member 의 oauthId 는 null 일 수 없습니다.");
+    private static void validateCompanyNotNull(final Company company) {
+        if (Objects.isNull(company)) {
+            throw new MemberDomainException("Member 의 company 는 null 일 수 없습니다.");
         }
+    }
 
+    private static void validateGithubUrlNotNull(final GithubUrl githubUrl) {
         if (Objects.isNull(githubUrl)) {
             throw new MemberDomainException("Member 의 githubUrl 은 null 일 수 없습니다.");
         }
+    }
 
-        if (Objects.isNull(company)) {
-            throw new MemberDomainException("Member 의 company 는 null 일 수 없습니다.");
+    private static void validateOauthIdNotNull(final OauthId oauthId) {
+        if (Objects.isNull(oauthId)) {
+            throw new MemberDomainException("Member 의 oauthId 는 null 일 수 없습니다.");
         }
+    }
 
-        if (Objects.isNull(imageUrl)) {
-            throw new MemberDomainException("Member 의 imageUrl 은 null 일 수 없습니다.");
+    private static void validateSocialIdNotNull(final SocialId socialId) {
+        if (Objects.isNull(socialId)) {
+            throw new MemberDomainException("Member 의 socialId 은 null 일 수 없습니다.");
         }
     }
+
+    private static void validateMemberNameNotNull(final MemberName memberName) {
+        if (Objects.isNull(memberName)) {
+            throw new MemberDomainException("Member 의 name 은 null 일 수 없습니다.");
+        }
+    }
+
+
+    public void updateMemberName(final MemberName memberName) {
+        validateMemberNameNotNull(memberName);
+        this.memberName = memberName;
+    }
+
+    public void updateCompany(final Company company) {
+        validateCompanyNotNull(company);
+        this.company = company;
+    }
```

Comment:
저희가 수정할 때 모든 값을 프론트에서 다 받는데
그러면 update 메서드를 하나하나 만들지 않고 updateMember 를 받는거는 어떤가요?

```suggestion
   public void updateMember(final Member updateMember) {
       this.memberName = updateMember.getMemberName();
       // ...
   }
```